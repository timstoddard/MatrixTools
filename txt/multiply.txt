2
3
1,2,3
3,2,1
3
4
1,2,3,4
5,6,6,5
4,3,2,1

3
2
4,8
0,2
1,6
2
2
5,2
9,4

3
3
6,3,0
2,5,1
9,8,6
3
2
7,4
6,7
5,0

1
3
3,4,2
3
4
13,9,7,15
8,7,4,6
6,4,0,3