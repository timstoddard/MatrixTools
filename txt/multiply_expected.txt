2
4
23,23,21,17
17,21,23,23

3
2
92,40
18,8
59,26

3
2
60,45
49,43
141,92

1
4
83,63,37,75